otherlibs/unix/unix.mli

```diff
@@ -394,16 +394,29 @@ val in_channel_of_descr : file_descr -> in_channel
    Text mode is supported only if the descriptor refers to a file
    or pipe, but is not supported if it refers to a socket.
 
-   On Windows: [set_binary_mode_in] always fails on channels created
-   with this function.
-
-   Beware that channels are buffered so more characters may have been
-   read from the file descriptor than those accessed using channel functions.
-   Channels also keep a copy of the current position in the file.
+   On Windows: {!Stdlib.set_binary_mode_in} always fails on channels
+   created with this function.
+
+   Beware that input channels are buffered, so more characters may
+   have been read from the descriptor than those accessed using
+   channel functions.  Channels also keep a copy of the current
+   position in the file.
+
+   Closing the channel [ic] returned by [in_channel_of_descr fd]
+   using [close_in ic] also closes the underlying descriptor [fd].
+   Closing both the channel [ic] and the descriptor [fd] is
+   incorrect and will result in a {!Stdlib.Sys_error} or {!Unix.error}
```

Comment:
Actually, this is an undefined behavior, since the file descriptor may be reused in between. We cannot guarantee thet there is an exception raised here.

Reply:
All right, I just put "It is incorrect to close both the channel [ic] and the descriptor [fd].".  Sometimes the less you write the better.